### PR TITLE
Adds duck-related content and a seclite

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

Duc ky

**Screenshots**

image

**Changelog**

:cl:
- add: Adds seclite and duck related content

### CODE REVIEW

File: Resources/Textures/Objects/Tools/seclite.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/52cbeaf94322209d4c6596818dda9fb3654729d3",
```
Comment (PaulRitter) [inline Resources/Textures/Objects/Tools/seclite.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/52cbeaf94322209d4c6596818dda9fb3654729d3 and modified by Swept",
```

File: Resources/Textures/Objects/Fun/ducky.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation and modified by Swept at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e",
```
Comment (PaulRitter) [inline Resources/Textures/Objects/Fun/ducky.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e and modified by Swept",
```

**Review by PaulRitter — APPROVED** [review]
nice work, just a nitpick

**Comment by mirrorcult** [comment]
Lovely

File: Resources/Prototypes/Entities/Clothing/Shoes/misc.yml [hunk 3]
```diff
@@ -31,6 +31,17 @@
   - type: Clothing
     sprite: Clothing/Shoes/Misc/slippers.rsi
 
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoeSlippersDuck
+  name: ducky slippers
+  description: You wish these made quacking sounds as you walked.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Shoes/misc.yml:38]: not merging without this feature....

Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Clothing/Shoes/misc.yml:38]: > 
> 
> not merging without this feature....

Fair, will work on this.

**Comment by SweptWasTaken** [comment]
@mirrorcult Wdyt

https://user-images.githubusercontent.com/49448379/130130931-3e782944-275f-494b-9b32-93e38dc09064.mp4

**Comment by mirrorcult** [comment]
speedmerge

**Comment by Ygg01** [comment]
> speedmerge

👍 
HyperNecessary

**Pull request merged**